src/model/psf_model.jl

```diff
@@ -1,5 +1,5 @@
-# The number of Gaussian components in the PSF.
-const psf_K = 2
+# The default number of Gaussian components in the PSF.
+const default_psf_K = 2
```

Comment:
By putting the default in the constructor to `ElboArgs`, there's no risk that code will call `default_psf_K` rather than `ea.psf_K`--potentially a difficult to detect bug.